pipeline/scripts/5/0/tier-3/test-rados-scrub-scenarios.sh

```diff
@@ -0,0 +1,48 @@
+#! /bin/sh
+echo "Beginning Ceph RADOS regression testing for Scrubbing scenarios"
+
+random_string=$(cat /dev/urandom | tr -cd 'a-z0-9' | head -c 5)
+instance_name="ci-${random_string}"
+platform="rhel-8"
+rhbuild="5.0"
+test_suite="suites/pacific/rados/tier_3_rados_scrub.yaml"
```

Comment:
Could we rename this suite file to `tier-3_rados_test-scrubbing.yaml`?

This allows us to have a proper consolidated report at the end.

Reply:
Renamed the suite file